gitversion/gitversion.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+import sys
+
+import time
+
+from pelican import signals
+
+
+
+logger = logging.getLogger(__name__)
+
+try:
+    from git import Repo
+except ImportError:
+    logger.warning("Could not import git!")
+    sys.exit(0)
```

Comment:
Uh. So... you're going to quit pelican completely because you can't import `git`? Isn't it enough to issue a warning?

PS: FWIW, this can be a simple function in the settings itself rather than a plugin, but OK I guess :).

Reply:
@avaris Thanks for the comment, I have removed the `sys.exit`